### DISCUSSION TITLE
Popup Cards in View missing

### DISCUSSION THREAD

**Discussion by darki79** [Clooos/Bubble-Card#1043] [Questions about config, custom styles and templates] [answered]
Since the last update my popup cards are all missing in my view, which were visible before:

<img width="513" alt="image" src="https://github.com/user-attachments/assets/e1bd26c6-ea6c-4769-8a24-78b5bd2e12d7" />

Yes, the buttons are on the bottom. When I edit this section via YAML I can see the whole code of this section:

<img width="641" alt="image" src="https://github.com/user-attachments/assets/e044b2ba-86c3-4863-9ff5-2aaeed118e95" />

Normally that happens, when there is an error in the yaml code, bit it worked till this morning (before the last update).

Could there be any changes that are now interpreted as error?

**Comment by MrBearPresident** [comment, marked as answer]
Hi

Due to some issues identified in the latest releases (v2.3.1, v2.3.2, v2.3.3), I have decided to mark them as pre-release versions. These updates may not be fully stable and could cause unexpected behavior. I strongly recommend rolling back to the latest stable release, **v2.3.0**, until further notice. This version has been thoroughly tested and provides a reliable experience. Thank you for your understanding and patience as we work to ensure a future updates that meets the highest standards of quality.

---

[u]**How to rollback**[/u]
1. Click on the integration  in HACS.
2. Click on the three dots icon top right of the page.
3. Select Redownload.
4. Choose the version you want from the dropdown list.
5. Click Download.

**Comment by darki79** [comment]
Thank you!
